lib/msf/core/exploit/remote/http/kubernetes/output/table.rb

```diff
@@ -0,0 +1,164 @@
+# -*- coding: binary -*-
+
+# Outputs Kubernetes API responses in table format
+class Msf::Exploit::Remote::HTTP::Kubernetes::Output::Table
+
+  # Creates a new `Msf::Exploit::Remote::HTTP::Kubernetes::Output::Table` instance
+  #
+  # @param [object] output The original output object with print_line/print_status/etc methods available.
+  # @param [String] highlight_name_pattern The regex used to highlight noteworthy resource names
+  def initialize(output, highlight_name_pattern: nil)
+    @output = output
+    @highlight_name_pattern = highlight_name_pattern
+  end
+
+  def print_error(*args)
+    output.print_error(*args)
+  end
+
+  def print_good(*args)
+    output.print_good(*args)
+  end
+
+  def print_status(*args)
+    output.print_status(*args)
+  end
+
+  def print_enum_failure(resource, error)
+    if error.is_a?(Msf::Exploit::Remote::HTTP::Kubernetes::Error::ApiError)
+      print_status("#{resource} failure - #{error.message}")
+    else
+      output.print_error(error.message)
+    end
+  end
+
+  def print_version(version)
+    print_good("Kubernetes service version: #{version.to_json}")
```

Comment:
It seems like when the table version is selected, that the output shouldn't be the raw JSON representation.

Reply:
At the time I intentionally did this to align with kubectl's default behavior, although it doesn't output JSON - just the stringified struct:

```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.2", GitCommit:"092fbfbf53427de67cac1e9fa54aaa09a28371d7", GitTreeState:"clean", BuildDate:"2021-06-16T12:59:11Z", GoVersion:"go1.16.5", Compiler:"gc", Platform:"darwin/amd64"}
```

Will update to be the more verbose table structure :+1:
```
Version
=======

  name          value
  ----          -----
  buildDate     2021-06-16T12:53:14Z
  compiler      gc
  gitCommit     092fbfbf53427de67cac1e9fa54aaa09a28371d7
  gitTreeState  clean
  gitVersion    v1.21.2
  goVersion     go1.16.5
  major         1
  minor         21
  platform      linux/amd64
```